bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ServerConfiguration.java

```diff
@@ -2188,6 +2189,29 @@ public long getAuditorPeriodicBookieCheckInterval() {
         return getLong(AUDITOR_PERIODIC_BOOKIE_CHECK_INTERVAL, 86400);
     }
 
+    /**
+     * Set the regularity at which the auditor will run a metadata check of all
+     * write closed/fenced ledgers. This should not be run very often, and at
```

Comment:
No need to say write closed -> as there is no read open / close ledger state. 
Also fenced ledger may not have final consistent state yet. Why do you need to include
fenced ledgers, it is just a transient state right?

Reply:
changed it to "all ledgers which are closed"